packages/core/src/components.d.ts

```diff
@@ -341,11 +341,11 @@ export namespace Components {
         "triggerAction"?: TriggerAction;
     }
     /**
-     * Describe whats does the component
+     * Dropdown can open a menu or any other element when the button is clicked.
```

Comment:
Dropdown can open a menu or any other element when the trigger element is clicked.